docker-compose.yaml

```diff
@@ -1,19 +1,47 @@
 version: '3'
 services:
-  sensu-backend1:
+  backend1:
     image: sensu/sensu:master
```

Comment:
Out of curiosity, why are we using `sensu/sensu:master` in some backends and agents, and `sensu/sensu-go:master` in others?

Reply:
Oh, that's a mistake, it should be sensu/sensu:master in all cases.

Reply:
No problem! I think it's just L14 and L24: https://github.com/sensu/sensu-go/blob/7dace965851e6f53f54c2b43dba102f6f649ff04/docker-compose.yaml#L14 and https://github.com/sensu/sensu-go/blob/7dace965851e6f53f54c2b43dba102f6f649ff04/docker-compose.yaml#L24